src/project/environment.rs

```diff
@@ -501,7 +500,54 @@ mod tests {
         );
         assert_eq!(
             foo_env.activation_scripts(Some(Platform::Linux64)),
-            vec!["foo.bat".to_string(), "linux.bat".to_string()]
+            vec![
+                "foo.bat".to_string(),
```

Comment:
I just remembered that we did this initially on purpose as you would have to define every target to use this.

Reply:
I've reverted the changes for activation scripts.